perception/autoware_traffic_light_classifier/README.md

```diff
@@ -43,20 +50,21 @@ These colors and shapes are assigned to the message as follows:
-| `data_path`           | str   | packages data and artifacts directory path                                                                                                                                                                                                                                                                                                                 |
-| `backlight_threshold` | float | If the intensity get grater than this overwrite with UNKNOWN in corresponding RoI. Note that, if the value is much higher, the node only overwrites in the harsher backlight situations. Therefore, If you wouldn't like to use this feature set this value to `1.0`. The value can be `[0.0, 1.0]`. The confidence of overwritten signal is set to `0.0`. |
+| Name                          | Type  | Description                                                                                                                                                                                                                                                                                                                                                |
+| ----------------------------- | ----- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `classifier_type`             | int   | if the value is `1`, cnn_classifier is used                                                                                                                                                                                                                                                                                                                |
+| `data_path`                   | str   | packages data and artifacts directory path                                                                                                                                                                                                                                                                                                                 |
+| `backlight_threshold`         | float | If the intensity get grater than this overwrite with UNKNOWN in corresponding RoI. Note that, if the value is much higher, the node only overwrites in the harsher backlight situations. Therefore, If you wouldn't like to use this feature set this value to `1.0`. The value can be `[0.0, 1.0]`. The confidence of overwritten signal is set to `0.0`. |
```

Comment:
Can you write this better please?

Reply:
I rewrote it as:https://github.com/autowarefoundation/autoware.universe/pull/6724/commits/e529247b56d761aa95e1c08828fc975a03d2d57b#diff-a6db2d5664399836c22f951aef589933ddf7d6ebf6077baf6616399984913404R66